ludwig/hyperopt/search_algos.py

```diff
@@ -30,19 +29,27 @@ def check_for_random_seed(self, ludwig_random_seed: int) -> None:
             self.search_alg_dict[self.random_seed_attribute_name] = ludwig_random_seed
 
 
+def get_search_algorithm_cls(name: str) -> Type[SearchAlgorithm]:
```

Comment:
What do you think about defining this along with the registry itself in the registry file and then importing it into this file. I feel like that might make a bit more organizational sense but I don't know. WDYT?

Reply:
done